app/src/main/java/ru/woodymsk/socialapp/data/event/EventRepositoryImpl.kt

```diff
@@ -44,11 +51,13 @@ class EventRepositoryImpl @Inject constructor(
                 )
             }
             if (!response.isSuccessful) response.body().throwAppError(response)
+            response.body()?.let { eventDao.insertEvent(eventMapper.mapDtoToEntity(it)) }
```

Comment:
```suggestion
            response.body()?.let { eventDao.insertEvent(eventMapper::mapDtoToEntity) }
```

Так красивее, мне кажется